Firebase/Messaging/FIRMessagingCodedInputStream.m

```diff
@@ -37,7 +37,7 @@ static BOOL ReadRawByte(BufferState *state, int8_t *output) {
   if (state == NULL || output == NULL) {
     FIRMessagingLoggerDebug(kFIRMessagingCodeInputStreamInvalidParameters, @"Invalid parameters.");
   }
```

Comment:
should we just return here like the way you did in the following lines?

Reply:
Probably, although different flow if output is not nil and state is nil.